exercises/change/change.spec.js

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
I'm glad all canonical cases are include, great!

I usually like more throwing errors than returning error codes in these exercises, so I'm ok with them too.

In this particular file, `change.spec.js`, I still see some indentation issues. Sorry to be so insistent with it. `xit` blocks should look like:

```javascript
  xit('test description', function () {  // indented two spaces compare to describe
    var change = new Change();           // indented two more spaces
    // ...
    expect(result).toEqual(...);
  });                                     // indented two spaces compare to describe
```